package.json

```diff
@@ -18,7 +18,7 @@
     "@react-leaflet/core": "^2.1.0",
     "@turf/turf": "^6.5.0",
     "acorn": "8.14.0",
-    "adhocracy4": "liqd/adhocracy4#48f1d9485acbed9a44e969ec1b741ad1c5469cf0",
+    "adhocracy4": "liqd/adhocracy4#main",
```

Comment:
we always use a hash here, so the outcome is predictable, with main it changes if something gets merged to a4